tests/testthat/testSel3D.R

```diff
@@ -0,0 +1,55 @@
+# testSel3D.R
+#
+#
+# Structure your tests according to the three principles expressed below:
+#    test_that("parameter errors are correctly handled", { ... })
+#    test_that("a sane input gives an expected output", { ... })
+#    test_that("a corrupt input does not lead to corrupted output", { ...})
+
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+#OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+#logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+#NL <- .PlatformLineBreak()
+# ==== END SETUP AND PREPARE ===================================================
+
+
+
+
+#test_that("parameter errors are correctly handled", {
+  # Try missing input sequence
+  # Try extraneous parameters
+  # Note: we're not to attempt to avoid all kinds of abuse from users
```

Comment:
In my opinion, if the parameters are just for recursion, you could:
1. Explicitly tell the user these parameters are preset and not meant to be changed in the preset argument descriptions. I think this option is "code smell" though.
2. Have getPdbIds set up a helper function that handles the recursive part. If the parameters aren't supposed to be changed, I think exposing them to the user should be avoided. 

What's your opinion?